test/Twig/Tests/EnvironmentTest.php

```diff
@@ -165,6 +173,9 @@ public function testExtensionsAreNotInitializedWhenRenderingACompiledTemplate()
         $this->assertEquals('bar', $output);
 
         unlink($cache);
+        if ($createdDir) {
+            rmdir($dir);
+        }
```

Comment:
Can you please revert all changes in this file. They are not related to the goal of this PR and controversial :)